Sources/XcodeProj/Objects/Files/PBXFileSystemSynchronizedRootGroup.swift

```diff
@@ -12,7 +12,7 @@ public class PBXFileSystemSynchronizedRootGroup: PBXFileElement {
 
     /// It returns a list of exception objects that override the configuration for some children
     /// in the synchronized root group.
-    public var exceptions: [PBXFileSystemSynchronizedBuildFileExceptionSet]? {
+    public var exceptions: [PBXFileSystemSynchronizedExceptionSet]? {
```

Comment:
This has two drawbacks:
- it's technically a breaking change. Given the breaking change is quite minor _and_ the `exceptions` are still quite new, I'm personally ok with this.
- To get build phase exception sets and build file exceptions (`PBXFileSystemSynchronizedGroupBuildPhaseMembershipExceptionSet` and `PBXFileSystemSynchronizedBuildFileExceptionSet`), one needs to typecast the individual elements. We _could_ make things more type-safe by splitting `exceptions` into `buildFileExceptions` and `buildPhaseMembershipExceptions` but we'd be going against how things are structured in the `.pbxproj`. I'm leaning to keeping things as-is, but can adjust this if somebody feels strongly that we should do this in a different way.